test/validate/eia_test.py

```diff
@@ -45,16 +45,16 @@ def test_no_null_cols_eia(pudl_out_eia, live_dbs, cols, df_name):
 @pytest.mark.parametrize(
     "df_name,raw_rows,monthly_rows,annual_rows",
     [
-        ("bf_eia923", 1_473_364, 1_473_364, 123_369),
+        ("bf_eia923", 1_521_340, 1_521_340, 127_415),
         ("bga_eia860", 141_638, 141_638, 141_638),
         ("boil_eia860", 80_059, 80_059, 80_059),
-        ("frc_eia923", 584_724, 241_751, 23_737),
-        ("gen_eia923", None, 4_765_282, 398_747),
+        ("frc_eia923", 632_035, 258_151, 25_373),
```

Comment:
These row counts should likely be a little higher than they were previous to merging the PPE because we're also harvesting owner utilities now and there are fewer null values in `utility_id_eia`.